tests/test_api.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase
+import pytest
+import yaml
+from operatorcourier import api
+
+@pytest.mark.parametrize('dir,expected', [
+("tests/test_files/bundles/api/bundle1", "tests/test_files/bundles/api/bundle1/results/bundle.yaml"),
+])
+def test_make_bundle(dir, expected):
+    bundle = api.build_and_verify(source_dir=dir)
```

Comment:
`dir` is python's built-in, please use different name for variable

Reply:
updated.